video/graphics.h

```diff
@@ -62,26 +62,29 @@ char getScreenChar(uint16_t px, uint16_t py) {
 	if (px >= canvasW - 8 || py >= canvasH - 8) {
 		return 0;
 	}
-
-	// Now scan the screen and get the 8 byte pixel representation in charData
-	//
-	for (uint8_t y = 0; y < 8; y++) {
-		charRow = 0;
-		for (uint8_t x = 0; x < 8; x++) {
-			pixel = canvas->getPixel(px + x, py + y);
-			if (!(pixel.R == R && pixel.G == G && pixel.B == B)) {
-				charRow |= (0x80 >> x);
-			}
-		}
-		charData[y] = charRow;
-	}
-	//
-	// Finally try and match with the character set array
-	//
-	for (auto i = 32; i <= 255; i++) {
-		if (cmpChar(charData, &fabgl::FONT_AGON_DATA[i * 8], 8)) {	
-			return i;		
-		}
+	if (ttxtMode) {
```

Comment:
minor comment...

this diff is a bit more complex than it should be because the VDP source files use tabs for indentation, rather than spaces